EDSAnalytics/Coverity.proj

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you copy the [analysis template](https://github.com/osisoft/OSI-Samples-OMF/blob/master/miscellaneous/build_templates/analysis.yml) used by the other projects into this repo and reference that, it should run Coverity (as well as BlackDuck/BinSkim) and this file shouldn't be necessary. (we are in the process of migrating to Polaris anyway)